notebooks/utils.py

```diff
@@ -0,0 +1,97 @@
+import json
+from typing import List
```

Comment:
let's use `list` instead of List here since we're on Python 3.11 https://peps.python.org/pep-0585/